virtaal/common/utils.py

```diff
@@ -0,0 +1,15 @@
+import sys
+
+from six import text_type
+
+
+def get_unicode(string, encoding=None):
+    if isinstance(string, text_type):
+        return string
+    return string.decode(encoding or sys.getfilesystemencoding())
+
+
+def get_bytes(string):
```

Comment:
This looks unused. Do you think this will be needed?

Reply:
These are used for python2 and python 3 compatible.It can be removed and fix the error where use it.